src/rulesets-new/lifecycle-rules.ts

```diff
@@ -0,0 +1,112 @@
+import { RuleError, Ruleset, SpecificationRule } from "@useoptic/rulesets-base";
+import { links } from "../docs";
+import { stabilityKey } from "./constants";
+
+const stabilityRequirement = new SpecificationRule({
+  name: "resource stability",
+  docsLink: links.versioning.stabilityLevels,
+  matches: (specification, rulesContext) =>
+    rulesContext.specification.change !== "removed",
+  rule: (specificationAssertions) => {
```

Comment:
This is updated that we now model stability on the specification object, rather than exposing a custom written stability assertion